users/curry/process_records.h

```diff
@@ -1,36 +1,28 @@
 #pragma once
-#include "curry.h"
```

Comment:
This is an example of where I removed a header file that was not needed for the compilation of this header file.